src/nodes/InheritanceSpecifier.js

```diff
@@ -1,21 +1,32 @@
 const {
   doc: {
-    builders: { concat }
+    builders: { concat, group, indent, join, line, softline }
   }
 } = require('prettier/standalone');
 
-const InheritanceSpecifier = {
-  print: ({ node, path, print }) => {
-    let parts = [path.call(print, 'baseName')];
+const printArguments = (node, path, print) => {
+  if (node.arguments && node.arguments.length) {
```

Comment:
I wonder what happens (and what should happen) if someone does `Foo is Bar()`. I think this will format it as `Foo is Bar`. I'm pretty sure that's OK, but just something to keep in mind.

Reply:
A quick check using https://astexplorer.net tells me that we will be removing the `()`.
This will generate a difference in the AST.
I believe that in this case we should be opinionated and remove the `()` anyway.
If we agree on this I'll add this rule so we can add this example and still pass the `AST_COMPARE=true` tests.

Reply:
Forget what I said.
`Foo is Bar` and `Foo is Bar()` create the same AST.
Thus we can't know from the AST alone.
I still believe that we should just ignore the `()`,  we could check the original source but that will be extra work that it's not really needed.

Reply:
+1 to removing it, especially if the AST is the same.

Reply:
I'll merge and deploy this one if that's ok